packages/opam-client/opam-client.2.2.0~alpha/opam

```diff
@@ -46,5 +46,4 @@ url {
     "sha512=b7899244ab620213f2997409c9d8ac63e1a7ba99ff36abdd0acf41ae545ca7e0534e5e2f6c01a9700c10abf2df4f07944cc64fb5dd315a566f6c5b6b19464729"
   ]
 }
-available: opam-version >= "2.1.0"
-flags: avoid-version
+flags: deprecated
```

Comment:
Won't this break opam 2.1 users if we remove the opam-version constraint as well?

Reply:
unknown flags are ignored so on opam 2.0 and 2.1 these packages will just not have any special behaviour.
In practice it shouldn't matter anymore as more recent versions are available. It was critical to have it at the time of release because it would've been the latest version and be picked up automatically by the solver, but this is not the case anymore